src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+
+import java.util.Date;
+
+public class Deadline extends Task {
+
+    protected Date dueDate;
+
+    public Deadline(String taskName, Date dueDate) {
+        super(taskName, false);
+        setBy(dueDate);
+    }
+
+    public void setBy(Date dueDate) {
```

Comment:
line 14 and  line 16 are getters and setters? If yes, consider naming them uniformly, getDueDate() and setDueDate() so that you know they are the "corresponding" methods that make use of the same variable dueDate